app/views/idv/in_person/ready_to_verify/show.html.erb

```diff
@@ -117,5 +117,5 @@
 <% end %>
 
 <%= render PageFooterComponent.new do %>
-  <%= link_to t('in_person_proofing.body.barcode.cancel_link_text'), idv_cancel_path(step: 'verify') %>
+  <%= link_to t('in_person_proofing.body.barcode.cancel_link_text'), idv_cancel_path(step: 'barcode') %>
```

Comment:
Why this change?

Reply:
there are 2 steps named `verify`, this change now makes sure each step has a unique name and allows us to reuse the `idv_start_over` and `idv_cancellation_go_back` events for this use case of arriving at the cancel page + taking an action from the barcode page rather than adding new events